packages/spaces/lib/format.js

```diff
@@ -13,7 +13,27 @@ module.exports = function () {
     return CIDR(cidrBlocks)
   }
 
-  function Status (s) {
+  function VPNStatus (s) {
+    let colored = s
+    switch (s) {
+      case 'UP':
+      case 'available':
+        colored = `${cli.color.green(colored)}`
+        break
+      case 'pending':
```

Comment:
Should we have `provisioning` and `deprovisioning` here?

Reply:
Done in 20c05f5